python-minimal/Makefile

```
@@ -0,0 +1,46 @@
+SHELL := /usr/local/bin/bash
+
+ECR_AWS_ACCOUNT ?= $(shell aws sts get-caller-identity --profile dp-ci --query Account --output text)
+ECR_AWS_ACCOUNT_URL = $(ECR_AWS_ACCOUNT).dkr.ecr.eu-west-2.amazonaws.com
+PYTHON_VERSION = 3.12
+ALPINE_VERSION = 3.20.3
+IMAGE_ALIAS = onsdigital/dp-concourse-tools-python-minimal:${ALPINE_VERSION}-python-${PYTHON_VERSION}
+
+.PHONY: build
+build:
+	docker build -t ${IMAGE_ALIAS} -f Dockerfile.alpine-python-minimal .
+
+	aws ecr get-login-password --region eu-west-2 --profile dp-ci | docker login --username AWS --password-stdin ${ECR_AWS_ACCOUNT_URL}
+
+	docker tag ${IMAGE_ALIAS} ${ECR_AWS_ACCOUNT_URL}/${IMAGE_ALIAS}
+
+	docker push ${ECR_AWS_ACCOUNT_URL}/${IMAGE_ALIAS}
```

```suggestion
.PHONY: publish
publish:
	docker push ${ECR_AWS_ACCOUNT_URL}/${IMAGE_ALIAS}
```

build should only build and not push, that should be a separate target

---

done